git_machete/cli.py

```diff
@@ -536,21 +536,20 @@ def launch(orig_args: List[str]) -> None:
 
         machete_client = MacheteClient(git)
 
-        if cmd != 'discover':
-            if not os.path.exists(machete_client.definition_file_path):
-                # We're opening in "append" and not "write" mode to avoid a race condition:
-                # if other process writes to the file between we check the
-                # result of `os.path.exists` and call `open`,
-                # then open(..., "w") would result in us clearing up the file
-                # contents, while open(..., "a") has no effect.
-                with open(machete_client.definition_file_path, "a"):
-                    pass
-            elif os.path.isdir(machete_client.definition_file_path):
-                # Extremely unlikely case, basically checking if anybody
-                # tampered with the repository.
-                raise MacheteException(
-                    f"{machete_client.definition_file_path} is a directory "
-                    "rather than a regular file, aborting")
+        if not os.path.exists(machete_client.definition_file_path):
```

Comment:
Hmmm so this means that machete file will be created before it's first written, also in case of `discover`... okay, shouldn't harm us